### PR TITLE
zarr support

### PR DESCRIPTION
**Pull request by ocefpaf**
[According to the docs](https://www.unidata.ucar.edu/software/netcdf/documentation/NUG/nczarr_head.html#nczarr_cmake) the defaults are:

```
The necessary CMake flags are as follows (with defaults)

-DENABLE_NCZARR=on – equivalent to the Automake –enable-nczarr option.
-DENABLE_NCZARR_S3=off – equivalent to the Automake –enable-nczarr-s3 option.
-DENABLE_NCZARR_S3_TESTS=off – equivalent to the Automake –enable-nczarr-s3-tests option.
```

I added those anyway to see if we can enable the feature. I'll turn the test on in the next commits.

### CODE REVIEW

**Comment by ocefpaf** [comment]
@rsignell-usgs and @kmuehlbauer, as I suspected the zarr support is enabled by default and it is working. See https://nbviewer.jupyter.org/urls/gist.githubusercontent.com/ocefpaf/5482c2a43fcdb811ea23dcb466a76420/raw/2afb9a1f6fc6a26e0894ce23f3c278f78772dec9/netcdf-c-zarr.ipynb

The zip files were not working though, we need to add libzip. Let's wait for https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/1418 to merge this one and get zip support. (In my gist above I have zip support b/c I used a local package built with it.)

**Comment by ocefpaf** [comment]
@isuruf I'm testing you suggestion but everything seems to be broken now. But I think it is unrelated :-/

**Comment by ocefpaf** [comment]
Do we need the quotes round the test name? It is still trying to run it :-/

File: recipe/build.sh [hunk 1]
```diff
@@ -95,25 +91,31 @@ cmake -DCMAKE_INSTALL_PREFIX=${PREFIX} \
...
-      -DCMAKE_C_FLAGS_DEBUG=${CFLAGS} \
       -DENABLE_CDF5=ON \
-      ${CMAKE_PLATFORM_FLAGS[@]} \
       ${PARALLEL} \
+      -DENABLE_NCZARR=on \
+      -DENABLE_NCZARR_S3=off \
+      -DENABLE_NCZARR_S3_TESTS=off \
       ${SRC_DIR}
 make install -j${CPU_COUNT} ${VERBOSE_CM}
 ctest -VV --output-on-failure -j${CPU_COUNT}
```
Comment (isuruf) [inline recipe/build.sh:101]: You need the skip here too

Comment (ocefpaf) [reply recipe/build.sh:101]: I guess we need to remove that line. It seems like the result of a bad merge.

**Comment by ocefpaf** [comment]
@dopplershift do you mind taking a look before we merge? If skipping that test is OK I'll trigger a pin update and a migration.

**Comment by ocefpaf** [comment]
@dopplershift and @kmuehlbauer note that this PR does not add the S3 features. We need to package https://github.com/aws/aws-sdk-cpp/tree/master/aws-cpp-sdk-s3 first to enable that.

**Comment by dopplershift** [comment]
@ocefpaf I'd like to see what #119 shows first.

File: recipe/build.sh [hunk 2]
```diff
@@ -95,25 +97,30 @@ cmake -DCMAKE_INSTALL_PREFIX=${PREFIX} \
...
-    done
 
-    # Fix build paths in nc-config
-    sed -i.bak "s#${BUILD_PREFIX}/bin/${CC}#${CC}#g" ${PREFIX}/bin/nc-config
-    rm ${PREFIX}/bin/nc-config.bak
+SKIP=""
+# Skip failing test on ppc74le
+if [[ ${target_platform} == "linux-ppc64le" ]]; then
+ SKIP="-E nc_test"
 fi
```
Comment (isuruf) [inline recipe/build.sh:112]:
```suggestion
```

**Pull request merged**